app/src/main/java/com/keylesspalace/tusky/components/compose/view/ProgressImageView.kt

```diff
@@ -68,8 +71,10 @@ class ProgressImageView
     }
 
     fun setChecked(checked: Boolean) {
-        markBgPaint.color =
-            context.getColor(if (checked) R.color.tusky_blue else R.color.tusky_grey_10)
+        val backgroundColor = if (checked) com.google.android.material.R.attr.colorPrimary else android.R.attr.colorBackground
```

Comment:
Would it make sense to import this somehow?

"com.google.android.material.R.attr.colorPrimary" looks a bit cumbersome (and lots of places, too)

Reply:
yes definitely, thank you